_includes/footer.html

```diff
@@ -2,6 +2,8 @@
   <ul>
     <li><a href="/about">About</a></li>
     <li><a href="/learn/municipal-bonds">Learn About Bonds</a></li>
+    <li><a href="http://www.kut.org/post/what-are-propositions-austin-voters-will-decide-november-its-alphabet-soup">KUT Article</a></li>
```

Comment:
I'm not a giant fan of "KUT Article" without context in the footer. Maybe "KUT's Proposition Coverage" or something? Or maybe we should have a single page for external links and the footer link should be, like, "Other Resources" or something?